lhotse/recipes/gigaspeech.py

```diff
@@ -0,0 +1,10 @@
+from speechcolab.datasets import gigaspeech
+
+
+def download_gigaspeech(
+        target_dir: Pathlike = '.',
+        dataset_parts: Optional[Union[str, Sequence[str]]] = "XS",
+        force_download: Optional[bool] = False
+) -> None:
```

Comment:
Can we handle the `speechcolab` dependency like the other optional deps, see e.g. the import of `librosa` here:

https://github.com/lhotse-speech/lhotse/blob/274cf876ee1e31ec8a837cbd5dc3b6f6bfdf2b79/lhotse/features/librosa_fbank.py#L85

The point is to try to import, if it fails then raise an exception and say "you can fix this with `pip install X`". Then we can remove the dep from `requirements.txt`.